spec/spec_helper.rb

```diff
@@ -113,7 +113,7 @@
 Capybara.configure do |config|
   config.run_server = false
   config.default_driver = :poltergeist
-  config.save_path = __dir__ + "/../tmp"
+  config.save_path = ENV["CAPYBARA_SAVE_PATH"] or (__dir__ + "/../tmp")
```

Comment:
@rubenarakelyan I'm still happy to change it if you want? I put the bracketing in from the start in an attempt to make it clear.